asyncauth.go

```diff
@@ -219,6 +219,12 @@ func (config *Config) renderInstructions(w http.ResponseWriter, req *http.Reques
 		}
 	}
 
+	// Get the Kommander Cluster CA
+	authCAData := ""
+	if config.Clusters[0].K8s_Ca_Pem != "" {
```

Comment:
Segmentation fault if `len(config.Clusters) < 1`? I see this in code that's not being updated by this PR, so not a new issue, but I'm curious.

Reply:
You are correct, this is me being lazy and pasting bad code, which I probably wrote. In my defence, this "should never happen", the application would bail before getting here if there were no clusters defined. But, if that behaviour changes... then :boom:.

Reply:
Ok, cool. Just checking :joy: